README.md

```diff
@@ -1,9 +1,14 @@
 <div align="center">
   <br>
-  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
+  <img alt="Open Sauced" src="https://github.com/open-sauced/assets/blob/main/logos/logo-on-dark.png">
```

Comment:
I'm not a fan of this since it makes the first screen you see in the readme super busy (between this, the "🍕 Pizza CLI 🍕" and the video) - a few suggestions:
- we could remove this
- add a banner asset like https://github.com/pocketbase/pocketbase (would probably need @isabensusan on that)

Reply:
Another good example of a README with a banner: https://github.com/gofiber/fiber

Reply:
I think the biggest issue here is that npm grabs what we have, so we wanted to have _something_. I'm fine with removing it though.